helpers/operator.go

```diff
@@ -74,3 +71,63 @@ func CreateTykOperatorSecret(clientset *kubernetes.Clientset) error {
 
 	return nil
 }
+
+func BootstrapTykEnterprisePortalSecret() error {
+	fmt.Println("Started bootstrapping enterprise portal secret")
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	secrets, err := clientset.CoreV1().Secrets(data.AppConfig.TykPodNamespace).
+		List(context.TODO(), v1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, value := range secrets.Items {
```

Comment:
Why are we getting list of secret and then deleting it?
Why can't we delete it directly?

Reply:
@andrei-tyk probably can answer this since I just copied what was there. But I bet that it has to do with the fact that we are running a request against the cluster and we are allowed to only list the secrets. 

https://github.com/TykTechnologies/tyk-helm-chart/pull/226/files#diff-ecf8f4839fbdaeb4e28f99af6c022251e1b98669a02263aa19a175f3590fcbd2R32

Reply:
I agree it's a bit inefficient (arguably it's only an extra List call and an iteration over a size-1 array at the moment) but I think we can solve it with an eventual PR given that we have already spent resources testing this PR. I can do a PR that will address all of the operator/bootstrapJobs/enterprisePortal inefficiencies.

Reply:
I think we can merge some secrets and clean this up as well. I do agree with @andrei-tyk this can be an improvement ticket.

Reply:
Sounds good